From b22df22583fef8b9100e239de6184af1f6863de5 Mon Sep 17 00:00:00 2001
From: waddlesplash <waddlesplash@gmail.com>
Date: Sun, 6 Mar 2016 12:42:53 -0500
Subject: [PATCH] mempatcher: Include unistd.h, as unlink(3) is used.

Some systems (e.g. Haiku) which have strict(er) POSIX implementations
require this.


diff --git a/src/mempatcher.cpp b/src/mempatcher.cpp
index 9995db2..fd3306b 100644
--- a/src/mempatcher.cpp
+++ b/src/mempatcher.cpp
@@ -31,6 +31,7 @@
 #include "mempatcher.h"
 #include "FileStream.h"
 #include "MemoryStream.h"
+#include <unistd.h>
 
 namespace Mednafen
 {
-- 
2.21.0
